Cargo.toml

```diff
@@ -46,13 +47,14 @@ maplit = "1.0.2"
 prometheus = "0.13.2"
 prost = "0.13.1"
 protobuf-src = "1.1.0"
-scylla = "0.13.0"
 serde = "1.0.145"
 serde_json = "1.0.86"
-solana-account-decoder = { git = "https://github.com/helius-labs/agave", branch = "master" }
-solana-logger = { git = "https://github.com/helius-labs/agave", branch = "master" }
-solana-sdk = { git = "https://github.com/helius-labs/agave", branch = "master" }
-solana-transaction-status = { git = "https://github.com/helius-labs/agave", branch = "master" }
+solana-account-decoder = "~2.1.2"
+solana-logger = "~2.1.2"
+solana-sdk = "~2.1.2"
+solana-transaction-status = "~2.1.2"
+solana-client = "~2.1.2"
+solana-rpc-client-api = "~2.1.2"
```

Comment:
Using the master git versions was giving me package conflict errors. So I will use the latest version of the release instead. I tested the code using the Solana test validator, and it worked.  So we should be good.

Reply:
This is risky, we should keep using the master versions since geyser usually has breaking changes between minor versions

Reply:
I retrospectively agree, but since this version has been battle tested with 2.2 and it's fixed, then I now prefer this over using the master version.